generators/artifacthub/error.go

```diff
@@ -26,3 +30,23 @@ func ErrGetAhPackage(err error) error {
 func ErrComponentGenerate(err error) error {
 	return errors.New(ErrComponentGenerateCode, errors.Alert, []string{"failed to generate components for the package"}, []string{err.Error()}, []string{}, []string{"Make sure that the package is compatible"})
 }
+func ErrChartUrlEmpty(modelName string) error {
+	return errors.New(
+		ErrChartUrlEmptyCode,
+		errors.Alert,
+		[]string{fmt.Sprintf("The Chart URL for the %s model is empty.", modelName)},
+		[]string{fmt.Sprintf("provided Chart URL for the model %s is empty.", modelName)},
+		[]string{fmt.Sprintf("Artifacthub does not have Chart URL for %s", modelName)},
```

Comment:
Use registrant name and do not be specific to Artifacthub

Reply:
Okay

Reply:
Still pending.